src/modules/producer.js

```diff
@@ -8,6 +8,7 @@ const ERRORS = {
   BUFFER_FULL: 'Buffer is full'
 };
 
+const bmqProducer = 'bmq:producer';
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS1192)